nnvm/src/top/nn/pooling.cc

```diff
@@ -44,23 +45,38 @@ inline bool Pool2DInferShape(const nnvm::NodeAttrs& attrs,
   const auto hidx = layout.indexof('H');
   const auto widx = layout.indexof('W');
 
+  TShape pad = param.padding;
+  if (param.padding.ndim() == 1) {
+    pad[1] = pad[0];
+    pad[2] = pad[0];
+    pad[3] = pad[0];
```

Comment:
the logic here seems not correct, it should be:
```python
pad_h = pad[0] * 2
pad_w = pad[0] * 2
```

Reply:
Good catch, thanks.